courses/free-courses-en.md

```diff
@@ -1604,6 +1604,7 @@
 * [Python for OSINT. 21 day course for beginners](https://github.com/cipher387/python-for-OSINT-21-days/raw/main/Python%20for%20OSINT.%2021%20day%20course%20for%20beginners.pdf) - cyb_detective (PDF)
 * [Python for Programmers](https://www.codecademy.com/learn/python-for-programmers) - Codecademy
 * [Python from Scratch](https://open.cs.uwaterloo.ca/python-from-scratch/) - Centre for Education in Math and Computing (University of Waterloo)
+* [Python Full Course for Beginners to Advanced | 12 Hours Complete Tutorial](https://www.youtube.com/watch?v=_aWbUudZ5Yo) - Sheryians AI School (YouTube)
```

Comment:
```suggestion
* [Python Full Course for Beginners to Advanced \| 12 Hours Complete Tutorial](https://www.youtube.com/watch?v=_aWbUudZ5Yo) - Sheryians AI School
```
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes